### ISSUE TITLE
[Select Component] Handle Custom Empty State Component When Search Not Found

### ISSUE THREAD

**Issue by rizaldirnm** [rizzui/rizzui#104]
is it possible to handle customize empty state component?

Thank you

![CleanShot 2024-11-14 at 21 43 51@2x](https://github.com/user-attachments/assets/546981e6-3295-402a-adda-8f1859ce7800)

**Comment by tesfustar** [comment]
yes i have seen this issue as well, i am intersted in fixing this issue if the owner assigned me

**Comment by md-nahid** [comment]
@rizaldirnm We’re currently working on the Select component and will be pushing an update soon. Stay tuned!
